### PR TITLE
show Limited Support status when calling `rosa describe cluster`

### PR DESCRIPTION
**Pull request by olisakov**
[SDA-6564](https://issues.redhat.com//browse/SDA-6564)
PTAL @oriAdler @tzvatot

### CODE REVIEW

**Comment by tzvatot** [comment]
@olisakov can you please attach an example here for the output?

File: cmd/describe/cluster/cmd.go [hunk 1]
```diff
@@ -291,7 +291,14 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 	// Print short cluster description:
 	fmt.Print(str)
-	fmt.Println()
+	limitedSupportReasons, err := r.OCMClient.GetLimitedSupportReasons(cluster.ID())
```
Comment (oriAdler) [inline cmd/describe/cluster/cmd.go:294]: NP: We can concatenate the limited support reasons to the `str` variable before line `293`.

**Comment by oriAdler** [comment]
LGTM

**Comment by olisakov** [comment]
~/workPlace/rosa (report-limited-support-status)! L $ ./rosa describe cluster -c oiskov-fake2
Name:                       oiskov-fake2
ID:                         1vmo9dvd1t1bqdkasipcchp0epj8196u
External ID:                fake-cluster-605a1de3-f1fb-492d-a94b-4cf12c816aec
Control Plane:              Customer hosted
OpenShift Version:          
Channel Group:              stable
DNS:                        oiskov-fake2.zv7x.i1.devshift.org
AWS Account:                765374464689
API URL:                    https://example.com/veryfakeapi
Console URL:                https://https://example.com/veryfakewebconsole
Region:                     us-east-1
Multi-AZ:                   false
Nodes:
 - Control plane:           3
 - Infra:                   2
 - Compute:                 2
Network:
 - Type:                    OVNKubernetes
 - Service CIDR:            172.30.0.0/16
 - Machine CIDR:            10.0.0.0/16
 - Pod CIDR:                10.128.0.0/14
 - Host Prefix:             /23
State:                      ready 
Private:                    No
Created:                    Oct 31 2022 15:50:54 IST
```
Limited Support:
 - Summary:                 ori
 - Details:                 olga
 - Summary:                 example
 - Details:                 blabla
```
@tzvatot

**Comment by olisakov** [comment]
```
Name:                       oiskov-fake2
ID:                         1vmo9dvd1t1bqdkasipcchp0epj8196u
External ID:                fake-cluster-605a1de3-f1fb-492d-a94b-4cf12c816aec
Control Plane:              Customer hosted
OpenShift Version:          
Channel Group:              stable
DNS:                        oiskov-fake2.zv7x.i1.devshift.org
AWS Account:                765374464689
API URL:                    https://example.com/veryfakeapi
Console URL:                https://https://example.com/veryfakewebconsole
Region:                     us-east-1
Multi-AZ:                   false
Nodes:
 - Control plane:           3
 - Infra:                   2
 - Compute:                 2
Network:
 - Type:                    OVNKubernetes
 - Service CIDR:            172.30.0.0/16
 - Machine CIDR:            10.0.0.0/16
 - Pod CIDR:                10.128.0.0/14
 - Host Prefix:             /23
State:                      ready 
Private:                    No
Created:                    Oct 31 2022 15:50:54 IST
Limited Support:
 - Summary:                 ori
 - Details:                 olga
 - Summary:                 example
 - Details:                 blabla
```

**Pull request merged**